mypyc/primitives/dict_ops.py

```diff
@@ -108,6 +108,19 @@ def emit_new_dict(emitter: EmitterInterface, args: List[str], dest: str) -> None
     error_kind=ERR_MAGIC,
     emit=emit_new_dict)
 
+dict_new_op = c_custom_op(
+    arg_types=[],
+    return_type=dict_rprimitive,
+    c_function_name='PyDict_New',
+    error_kind=ERR_MAGIC)
+
+dict_build_op = c_custom_op(
```

Comment:
Add a single-line comment that documents what this does and explains the arguments.